shared/middleware/helpers/bucket_helper.rb

```diff
@@ -85,6 +85,7 @@ def list(encrypted_channel_id)
   end
 
   def get(encrypted_channel_id, filename, if_modified_since = nil, version = nil)
+    if_modified_since = nil if if_modified_since == ''
```

Comment:
Addressing the same issue as https://github.com/code-dot-org/code-dot-org/pull/21523#discussion_r177920370.